types/algoliasearch/index.d.ts

```diff
@@ -1222,113 +1224,113 @@ declare namespace algoliasearch {
      * default:
-     * https://github.com/algolia/algoliasearch-client-js#allowtyposonnumerictokens
+     * https://www.algolia.com/doc/api-reference/api-parameters/allowTyposOnNumericTokens/
      */
     allowTyposOnNumericTokens?: boolean;
     /**
      * If set to true, plural won't be considered as a typo
      * default: false
-     * https://github.com/algolia/algoliasearch-client-js#ignoreplurals
+     * https://www.algolia.com/doc/api-reference/api-parameters/ignorePlurals/
      */
     ignorePlurals?: boolean;
     /**
      * List of attributes on which you want to disable typo tolerance
-     * default: ""
-     * https://github.com/algolia/algoliasearch-client-js#disabletypotoleranceonattributes
+     * default: []
+     * https://www.algolia.com/doc/api-reference/api-parameters/disableTypoToleranceOnAttributes/
      */
-    disableTypoToleranceOnAttributes?: string;
+    disableTypoToleranceOnAttributes?: string[];
     /**
      * Search for entries around a given location
      * default: ""
-     * https://github.com/algolia/algoliasearch-client-js#aroundlatlng
+     * https://www.algolia.com/doc/api-reference/api-parameters/aroundLatLng/
      */
     aroundLatLng?: string;
     /**
      * Search for entries around a given latitude/longitude automatically computed from user IP address.
      * default: ""
-     * https://github.com/algolia/algoliasearch-client-js#aroundlatlngviaip
+     * https://www.algolia.com/doc/api-reference/api-parameters/aroundLatLngViaIP/
      */
     aroundLatLngViaIP?: string;
     /**
      * Control the radius associated with a geo search. Defined in meters.
      * default: null
      * You can specify aroundRadius=all if you want to compute the geo distance without filtering in a geo area
-     * https://github.com/algolia/algoliasearch-client-js#aroundradius
+     * https://www.algolia.com/doc/api-reference/api-parameters/aroundRadius/
      */
     aroundRadius?: number | 'all';
     /**
      * Control the precision of a geo search
      * default: null
-     * https://github.com/algolia/algoliasearch-client-js#aroundprecision
+     * https://www.algolia.com/doc/api-reference/api-parameters/aroundPrecision/
      */
     aroundPrecision?: number;
     /**
      * Define the minimum radius used for a geo search when aroundRadius is not set.
      * default: null
-     * https://github.com/algolia/algoliasearch-client-js#minimumaroundradius
+     * https://www.algolia.com/doc/api-reference/api-parameters/minimumAroundRadius/
      */
     minimumAroundRadius?: number;
     /**
      * Search entries inside a given area defined by the two extreme points of a rectangle
      * default: null
-     * https://github.com/algolia/algoliasearch-client-js#insideboundingbox
+     * https://www.algolia.com/doc/api-reference/api-parameters/insideBoundingBox/
      */
-    insideBoundingBox?: number[][];
+    insideBoundingBox?: Array<number|number[]>;
```

Comment:
Yes, this is more correct, since we accept both forms. I'd also be okay with leaving it as is and only accepting the nested array form, since the only reason the flat array form exist is because of earlier limitations of buggy JSON serialisation.

Reply:
Thanks, I've reverted this line back to the previous version.